modules/images/regenerate-existing-images/background-process/class-perflab-background-job.php

```diff
@@ -0,0 +1,236 @@
+<?php
+/**
+ * Background job class.
+ *
+ * @since n.e.x.t
+ * @package performance-lab
+ */
+
+/**
+ * Class Perflab_Background_Job.
+ *
+ * Runs the heavy lifting tasks in background in separate process.
+ */
+final class Perflab_Background_Job {
+	/**
+	 * Meta key for storing job name/action.
+	 *
+	 * @const string
```

Comment:
Add `@since n.e.x.t` for all constant and variables https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#2-1-class-members